src/Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/Bjorhus.cpp

```diff
@@ -239,6 +243,13 @@ std::optional<std::string> ConstraintPreservingBjorhus<Dim>::dg_time_derivative(
       make_not_null(&bc_dt_v_zero), unit_interface_normal_vector,
       four_index_constraint, char_speeds);
 
+  // unconditionally freeze Vplus, where needed
+  for (size_t a = 0; a <= Dim; ++a) {
+    for (size_t b = a; b <= Dim; ++b) {
+      bc_dt_v_plus.get(a, b) = -char_projected_rhs_dt_v_plus.get(a, b);
```

Comment:
Please add a note on this to the documentation of the class.

Also, I don't quite understand why you set the time derivative to the negative dt_v_plus. I would have expected it be set to zero. I'm sure this is because I don't know the interface for the boundary conditions well enough, but I wouldn't know where to look for this documentation. Please add a comment here, or refer to some docs, or something like that.

Reply:
Yeah, that is a good question. These classes are supposed to return the correction term that is then added to the pre-existing time derivative, i.e. 

`final_dt_vars = existing_dt_vars + corrs_from_this_class`

So in order to set the RHS to zero, this class must return `corrs_from_this_class = -existing_dt_vars`. This is also why, for e.g., the constraint preserving conditions on `VMinus` are assigned `-existing_dt_vars + intended_final_dt_vars` so as to make the `final_dt_vars = intended_final_dt_vars` [here](https://github.com/sxs-collaboration/spectre/blob/e77903ddfa844335d80d7b2d36dcac34d1ced6e1/src/Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/BjorhusImpl.cpp#L530)

I've added an inline comment here, and squashed the fixup.

Reply:
Thanks! Please also add a note on freezing V+ to the documentation of the class.

Reply:
done